helm/cluster-openstack/templates/_machine_health_check.tpl

```diff
@@ -0,0 +1,22 @@
+{{- define "machine-health-check" }}
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: MachineHealthCheck
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: {{ include "resource.default.name" . }}
+  namespace: {{ $.Release.Namespace }}
+spec:
+  clusterName: {{ include "resource.default.name" $ }}
+  selector:
+    matchExpressions:
+      - key: cluster.x-k8s.io/control-plane
+        operator: DoesNotExist
+  unhealthyConditions:
+  - type: Ready
+    status: Unknown
+    timeout: 300s
+  - type: Ready
+    status: "False"
+    timeout: 300s
```

Comment:
Five minutes feels a little short to me. It might be nice to review our rules for existing health checks on other providers to make sure we choose a reasonable number here.